composure.sh

```diff
@@ -379,7 +383,7 @@ metafor ()
   # 'grep' for the metadata keyword, and then parse/filter the matching line
 
   # grep keyword # strip trailing '|"|; # ignore thru keyword and leading '|"
-  sed -n "/$keyword / s/['\";]*\$//;s/^[ 	]*$keyword ['\"]*\([^([].*\)*\$/\1/p"
+  sed -n "/$keyword / s/['\";]*\$//;s/^[ 	]*\(: _\)*$keyword ['\"]*\([^([].*\)*\$/\2/p"
```

Comment:
The `: _` syntax is clever, @gaelicWizard! If I'd thought of it, I might not have implemented `cite()` to begin with. At the cost of some slight line noise it removes the need to cite metadata functions altogether, since`: _mymeta` will *always* be greppable.

I think I'd prefer that you revert the change on [line 278](https://github.com/erichs/composure/pull/30/files#diff-d2cd33bb09d42e95c4720656d08b7565be66634699678ec93ca9ef0593248145R278), and leave `cite()` to support the "porcelain" metadata syntax, meanwhile support for the "plumbing" metadata syntax you've introduced `: _` will JustWork:tm:.

Reply:
After this change, I'll document this behavior in the README: custom porcelain metadata requires `cite()`ation, but custom plumbing metadata does not.

Reply:
Done and done!